src/pages/api/clinical-trial-search.ts

```diff
@@ -1,8 +1,150 @@
 import type { NextApiRequest, NextApiResponse } from 'next';
-import mockSearchResults from '@/__mocks__/results.json';
+import { SearchParameters } from '@/utils/search_types';
+import { Bundle, BundleEntry, Resource } from '@/utils/fhir-types';
 
-const handler = (req: NextApiRequest, res: NextApiResponse<typeof mockSearchResults>): void => {
-  res.status(200).json(mockSearchResults);
+// Matching services and their information
+const services = {
+  breastCancerTrials: {
+    service_name: 'Breast Cancer Trials',
+    url: 'http://localhost:3001',
+    search_route: '/getClinicalTrial',
+  },
+  trialjectory: { service_name: 'TrialJectory', url: 'http://localhost:3000', search_route: '/getClinicalTrial' },
+  trialscope: { service_name: 'TrialScope', url: 'http://localhost:3000', search_route: '/getClinicalTrial' },
```

Comment:
Use camel case instead of snake case, which is the js standard. So `service_name` and `search_route` should be `serviceName` and `searchRoute`.